### PR TITLE
Add Pi-hole setup instructions

### PR DESCRIPTION
**Pull request by 2secslater**

### CODE REVIEW

**Comment by DaMatrix** [comment]
Shouldn't it be enough to simply add the txt file as a blocklist? I have it set up like so:

image

**Comment by 2secslater** [comment]
Ah yes, that works too. I forgot that existed. I clearly overcomplicated things, huh.

**Comment by 2secslater** [comment]
Actually, no. This does not work, @DaMatrix . 
1. The blocklist you are trying to retreive is not recognised as an adblock or hosts list.
2. The IP addresses are stripped, leaving only the domains in the "blocklist". It's in the name, these domains are going to be blocked by your Pi-hole if you have that list added.

So, I think my method that I am trying to merge is actually perfectly fine and functional.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# RiiConnect24 DNS Server [![Actions Status](https://github.com/RiiConnect24/DNS-Server/workflows/CI/badge.svg)](https://github.com/RiiConnect24/DNS-Server/actions)
+RiiConnect24 DNS Server
+===
+[![Actions Status](https://github.com/RiiConnect24/DNS-Server/workflows/CI/badge.svg)](https://github.com/RiiConnect24/DNS-Server/actions)
```
Comment (Artuto) [inline README.md]: Perhaps revert this? Useless to have an empty line only for the badge

Comment (dhtdht020) [reply README.md]: Additionally please change `/CI/badge.svg` to `/Build/badge.svg`, this causes a conflict as a result of recent commit.

**Comment by 2secslater** [comment]
Alrighty, I've reverted the badge move and updated the URL for it.

**Pull request merged**